apps/meteor/server/settings/omnichannel.ts

```diff
@@ -632,6 +632,21 @@ export const createOmniSettings = () =>
 			i18nLabel: 'Call_provider',
 			enableQuery: omnichannelEnabledQuery,
 		});
+
+		await this.add('Livechat_hide_system_messages', ['uj', 'ul', 'livechat-close', 'livechat_webrtc_video_call'], {
```

Comment:
How did we decide to cover these settings only? Are there more omnichannel system messages right? What should happen to the other cases?

Reply:
This setting values were decided based on the system messages the widget is currently capable of rendering.
https://github.com/RocketChat/Rocket.Chat/blob/201e97aba39afc30a8fd15d0bc98328b84693fb7/packages/livechat/src/components/Messages/Message/index.js#L67

It only covers system messages relevant to the Livechat rooms. Any system message received will either be rendered or ignored.

Reply:
Got it! Thanks.